src/main/java/com/allocadia/carbonite/CarbonRowMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.allocadia.carbonite;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+
+import lombok.SneakyThrows;
+
+public class CarbonRowMapper<T> implements RowMapper<T> {
```

Comment:
Yeah, or some combo of those, but with consistent meaning.  The library is Carbonite, things modified/wrapped to work with the library are Corbonated? idk... I can change it to whatever if you have a preference...